lib/classes/Swift/Transport/MailTransport.php

```diff
@@ -239,6 +239,35 @@ private function _getReversePath(Swift_Mime_Message $message)
     }
 
     /**
+     * Fix CVE-2016-10074 by disallowing potentially unsafe shell characters.
+     *
+     * Note that escapeshellarg and escapeshellcmd are inadequate for our purposes, especially on Windows.
+     * @param string $string The string to be validated
+     * @see https://github.com/swiftmailer/swiftmailer/issues/844
+     * @return boolean
+     */
```

Comment:
We need to ask permission to reuse this piece of code as PHPMailer is released under the GPL license, whereas we are using MIT for Swiftmailer.